_docs_integrate/error-codes.md

```diff
@@ -24,74 +24,159 @@ required_by:
-| {% include anchor a="error.transport.datawallet.encryptedPayloadIsNoCipher" %}                              | The given encrypted payload is no cipher.                                                                                                                                                                                                                                 |
-| {% include anchor a="error.transport.datawallet.insufficientSupportedDatawalletVersion" %}                  | The current SupportedDatawalletVersion is too low to perform the requested operation.                                                                                                                                                                                     |
-| {% include anchor a="error.transport.datawallet.unsupportedModification" %}                                 | A collection was recieved in a CacheChanged datawallet modification but is not supported in the current version of the library.                                                                                                                                           |
-| {% include anchor a="error.transport.devices.alreadyOnboarded" %}                                           | The device that you try to create an onboarding code for has already been onboarded.                                                                                                                                                                                      |
-| {% include anchor a="error.transport.files.cipherMismatch" %}                                               | The actual hash of the cipher does not match the given cipherHash. Something went wrong while storing/transmitting the file.                                                                                                                                              |
-| {% include anchor a="error.transport.files.fileContentUndefined" %}                                         | The given files content is undefined.                                                                                                                                                                                                                                     |
-| {% include anchor a="error.transport.files.invalidMetadata" %}                                              | The metadata of the file is invalid.                                                                                                                                                                                                                                      |
```

Comment:
This error message is still used, so please keep it in the documentation.

Reply:
I wanted to delete fileContentUndefined ...